packages/interaction/src/InteractionManager.js

```diff
@@ -939,7 +939,9 @@ export default class InteractionManager extends EventEmitter
      */
     dispatchEvent(displayObject, eventString, eventData)
     {
-        if (!eventData.stopped)
+        // Even if the event was stopped, at least dispatch any remaining events
+        // for the same display object.
+        if (!eventData.stopPropagationHint || displayObject === eventData.stopsPropagatingAt)
```

Comment:
This is where stuff went wrong. Because we just checked ``event.stopped`` here, a following event on the same object would also get killed. So now we also check whether the current object is still the same we wanted to stop propagation at.

Additionally we now use a separate ``stopPropagationHint``. This is because with delayed events we have to start over deeper in the tree, which is why we set it to false and then back to tru once we reach ``stopPropagationAt``.
This allows us to keep ``stopped`` clean and behaving the way users are used to.